rialto_airflow/utils.py

```diff
@@ -44,6 +44,9 @@ def invert_dict(dict):
 
 
```

Comment:
Not all publications have a DOI, but we still want to store them in the Publications table. It seemed easier to just let this function return None when passed a None?